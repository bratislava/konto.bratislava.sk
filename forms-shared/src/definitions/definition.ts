import { Schemas } from '../generator/functions'
import stanoviskoKInvesticnemuZameru from './stanovisko-k-investicnemu-zameru'
import predzahradky from './predzahradky'
import priznanieKDaniZNehnutelnosti from './priznanie-k-dani-z-nehnutelnosti'
import { generalTermsAndConditions, taxTermsAndConditions } from './terms-and-conditions'
import zavazneStanoviskoKInvesticnejCinnosti from './zavazne-stanovisko-k-investicnej-cinnosti'
import komunitneZahrady from './komunitne-zahrady'

enum FormDefinitionType {
  SlovenskoSk = 'SlovenskoSk',
  Tax = 'Tax',
  Email = 'Email',
}

type FormDefinitionBase = {
  slug: string
  schemas: Schemas
  title: string
  termsAndConditions: string
}

type FormDefinitionSlovenskoSk = FormDefinitionBase & {
  type: FormDefinitionType.SlovenskoSk | FormDefinitionType.Tax
  pospID: string
  pospVersion: string
}

type FormDefinitionEmail = FormDefinitionBase & {
  type: FormDefinitionType.Email
  email: string
}

type FormDefinition = FormDefinitionSlovenskoSk | FormDefinitionEmail

export const formDefinitions: FormDefinition[] = [
  {
    type: FormDefinitionType.SlovenskoSk,
    title: 'Žiadosť o stanovisko k investičnému zámeru',
    slug: 'stanovisko-k-investicnemu-zameru',
    schemas: stanoviskoKInvesticnemuZameru,
    pospID: '00603481.zavazneStanoviskoKInvesticnejCinnosti',
    pospVersion: '0.7',
    termsAndConditions: generalTermsAndConditions,
  },
  {
    type: FormDefinitionType.SlovenskoSk,
    title: 'Žiadosť o záväzné stanovisko k investičnej činnosti',
    slug: 'zavazne-stanovisko-k-investicnej-cinnosti',
    schemas: zavazneStanoviskoKInvesticnejCinnosti,
    pospID: '00603481.stanoviskoKInvesticnemuZameru',
    pospVersion: '0.7',
    termsAndConditions: generalTermsAndConditions,
  },
  {
    type: FormDefinitionType.SlovenskoSk,
    title: 'Predzáhradky',
    slug: 'predzahradky',
    schemas: predzahradky,
    pospID: '00603481.predzahradky',
    pospVersion: '1.0',
    termsAndConditions: generalTermsAndConditions,
  },
  {
    type: FormDefinitionType.SlovenskoSk,
    title: 'Komunitné záhrady',
    slug: 'predzahradky',
    schemas: komunitneZahrady,
    pospID: '00603481.komunitneZahrady',
    pospVersion: '1.0',
    termsAndConditions: generalTermsAndConditions,
  },
  {
    type: FormDefinitionType.Tax,
    title: 'Priznanie k dani z nehnuteľností',
    slug: 'priznanie-k-dani-z-nehnutelnosti',
    schemas: priznanieKDaniZNehnutelnosti,
    pospID: 'esmao.eforms.bratislava.obec_024',
    pospVersion: '201501.2',
    termsAndConditions: taxTermsAndConditions,
  },
]
