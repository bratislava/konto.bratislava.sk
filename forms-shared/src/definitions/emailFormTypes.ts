export enum MailgunTemplateEnum {
  NASES_SENT = 'NASES_SENT',
  GINIS_DELIVERED = 'GINIS_DELIVERED',
  NASES_GINIS_IN_PROGRESS = 'NASES_GINIS_IN_PROGRESS',
  GINIS_SUCCESS = 'GINIS_SUCCESS',
  GINIS_DENIED = 'GINIS_DENIED',
  ATTACHMENT_VIRUS = 'ATTACHMENT_VIRUS',
  OLO_NEW_SUBMISSION = 'OLO_NEW_SUBMISSION',
  OLO_SENT_SUCCESS = 'OLO_SENT_SUCCESS',
  TSB_NEW_SUBMISSION = 'TSB_NEW_SUBMISSION',
  TSB_SENT_SUCCESS = 'TSB_SENT_SUCCESS',
}
