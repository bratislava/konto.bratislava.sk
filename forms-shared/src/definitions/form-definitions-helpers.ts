import {
  FormDefinition,
  FormDefinitionEmail,
  formDefinitions,
  FormDefinitionSlovenskoSk,
  FormDefinitionSlovenskoSkGeneric,
  FormDefinitionSlovenskoSkTax,
  FormDefinitionType,
} from './form-definitions'

export const isSlovenskoSkGenericFormDefinition = (
  formDefinition: FormDefinition,
): formDefinition is FormDefinitionSlovenskoSkGeneric =>
  formDefinition.type === FormDefinitionType.SlovenskoSkGeneric

export const isSlovenskoSkTaxFormDefinition = (
  formDefinition: FormDefinition,
): formDefinition is FormDefinitionSlovenskoSkTax =>
  formDefinition.type === FormDefinitionType.SlovenskoSkTax

export const isSlovenskoSkFormDefinition = (
  formDefinition: FormDefinition,
): formDefinition is FormDefinitionSlovenskoSk =>
  isSlovenskoSkGenericFormDefinition(formDefinition) ||
  isSlovenskoSkTaxFormDefinition(formDefinition)

export const isEmailFormDefinition = (
  formDefinition: FormDefinition,
): formDefinition is FormDefinitionEmail => formDefinition.type === FormDefinitionType.Email

export const getFormDefinitionBySlug = (slug: string): FormDefinition | null => {
  const formDefinition = formDefinitions.find((formDefinition) => formDefinition.slug === slug)
  return formDefinition ?? null
}
