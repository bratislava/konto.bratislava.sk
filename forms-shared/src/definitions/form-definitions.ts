import { Schemas } from '../generator/functions'
import stanoviskoKInvesticnemuZameru from './stanovisko-k-investicnemu-zameru'
import predzahradky from './predzahradky'
import priznanieKDaniZNehnutelnosti from './priznanie-k-dani-z-nehnutelnosti'
import { generalTermsAndConditions, taxTermsAndConditions } from './terms-and-conditions'
import zavazneStanoviskoKInvesticnejCinnosti from './zavazne-stanovisko-k-investicnej-cinnosti'
import komunitneZahrady from './komunitne-zahrady'

enum FormDefinitionType {
  SlovenskoSk = 'SlovenskoSk',
  Tax = 'Tax',
  Email = 'Email',
}

type FormDefinitionBase = {
  slug: string
  title: string
  schemas: Schemas
  termsAndConditions: string
}

type FormDefinitionSlovenskoSk = FormDefinitionBase & {
  type: FormDefinitionType.SlovenskoSk | FormDefinitionType.Tax
  pospID: string
  pospVersion: string
}

type FormDefinitionEmail = FormDefinitionBase & {
  type: FormDefinitionType.Email
  email: string
}

type FormDefinition = FormDefinitionSlovenskoSk | FormDefinitionEmail

export const formDefinitions: FormDefinition[] = [
  {
    type: FormDefinitionType.SlovenskoSk,
    slug: 'stanovisko-k-investicnemu-zameru',
    title: 'Žiadosť o stanovisko k investičnému zámeru',
    schemas: stanoviskoKInvesticnemuZameru,
    pospID: '00603481.zavazneStanoviskoKInvesticnejCinnosti',
    pospVersion: '0.7',
    termsAndConditions: generalTermsAndConditions,
  },
  {
    type: FormDefinitionType.SlovenskoSk,
    slug: 'zavazne-stanovisko-k-investicnej-cinnosti',
    title: 'Žiadosť o záväzné stanovisko k investičnej činnosti',
    schemas: zavazneStanoviskoKInvesticnejCinnosti,
    pospID: '00603481.stanoviskoKInvesticnemuZameru',
    pospVersion: '0.7',
    termsAndConditions: generalTermsAndConditions,
  },
  {
    type: FormDefinitionType.SlovenskoSk,
    slug: 'predzahradky',
    title: 'Predzáhradky',
    schemas: predzahradky,
    pospID: '00603481.predzahradky',
    pospVersion: '1.0',
    termsAndConditions: generalTermsAndConditions,
  },
  {
    type: FormDefinitionType.SlovenskoSk,
    slug: 'komunitne-zahrady',
    title: 'Komunitné záhrady',
    schemas: komunitneZahrady,
    pospID: '00603481.komunitneZahrady',
    pospVersion: '1.0',
    termsAndConditions: generalTermsAndConditions,
  },
  {
    type: FormDefinitionType.Tax,
    title: 'Priznanie k dani z nehnuteľností',
    slug: 'priznanie-k-dani-z-nehnutelnosti',
    schemas: priznanieKDaniZNehnutelnosti,
    pospID: 'esmao.eforms.bratislava.obec_024',
    pospVersion: '201501.2',
    termsAndConditions: taxTermsAndConditions,
  },
]
