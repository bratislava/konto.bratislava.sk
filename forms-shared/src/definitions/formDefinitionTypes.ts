import { Schemas } from '../generator/functions'

export enum FormDefinitionType {
  SlovenskoSkGeneric = 'SlovenskoSkGeneric',
  SlovenskoSkTax = 'SlovenskoSkTax',
  Email = 'Email',
}

type FormDefinitionBase = {
  slug: string
  title: string
  schemas: Schemas
  termsAndConditions: string
  messageSubjectDefault: string
  messageSubjectFormat?: string
}

type FormDefinitionSlovenskoSkBase = FormDefinitionBase & {
  slovenskoSkUrl: string
  pospID: string
  pospVersion: string
  gestor: string
  isSigned: boolean
}

export type FormDefinitionSlovenskoSkGeneric = FormDefinitionSlovenskoSkBase & {
  type: FormDefinitionType.SlovenskoSkGeneric
  ginisAssignment: {
    ginisOrganizationName: string
    ginisPersonName: string
  }
}
export type FormDefinitionSlovenskoSkTax = FormDefinitionSlovenskoSkBase & {
  type: FormDefinitionType.SlovenskoSkTax
}

export type FormDefinitionSlovenskoSk =
  | FormDefinitionSlovenskoSkGeneric
  | FormDefinitionSlovenskoSkTax

export type FormDefinitionEmail = FormDefinitionBase & {
  type: FormDefinitionType.Email
  email: string
}

export type FormDefinition =
  | FormDefinitionSlovenskoSkGeneric
  | FormDefinitionSlovenskoSkTax
  | FormDefinitionEmail

export const isSlovenskoSkGenericFormDefinition = (
  formDefinition: FormDefinition,
): formDefinition is FormDefinitionSlovenskoSkGeneric =>
  formDefinition.type === FormDefinitionType.SlovenskoSkGeneric

export const isSlovenskoSkTaxFormDefinition = (
  formDefinition: FormDefinition,
): formDefinition is FormDefinitionSlovenskoSkTax =>
  formDefinition.type === FormDefinitionType.SlovenskoSkTax

export const isSlovenskoSkFormDefinition = (
  formDefinition: FormDefinition,
): formDefinition is FormDefinitionSlovenskoSk =>
  isSlovenskoSkGenericFormDefinition(formDefinition) ||
  isSlovenskoSkTaxFormDefinition(formDefinition)

export const isEmailFormDefinition = (
  formDefinition: FormDefinition,
): formDefinition is FormDefinitionEmail => formDefinition.type === FormDefinitionType.Email
