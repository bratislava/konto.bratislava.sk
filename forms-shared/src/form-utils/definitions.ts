import { FormDefinition, formDefinitions } from "../definitions/form-definitions";

export const getFormDefinitionBySlug = <T extends FormDefinition>(slug: string): T | null => {
  const formDefinition = formDefinitions.find(formDefinition => formDefinition.slug === slug)
  return formDefinition as T ?? null
}