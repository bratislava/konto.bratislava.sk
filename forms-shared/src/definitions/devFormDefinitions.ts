import { FormDefinition } from './formDefinitionTypes'

export const devFormDefinitions: FormDefinition[] = []
