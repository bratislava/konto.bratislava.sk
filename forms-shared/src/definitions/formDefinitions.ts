import stanoviskoKInvesticnemuZameru from '../schemas/stanoviskoKInvesticnemuZameru'
import predzahradky from '../schemas/predzahradky'
import priznanieKDaniZNehnutelnosti from '../schemas/priznanieKDaniZNehnutelnosti'
import { generalTermsAndConditions, taxTermsAndConditions } from './termsAndConditions'
import zavazneStanoviskoKInvesticnejCinnosti from '../schemas/zavazneStanoviskoKInvesticnejCinnosti'
import komunitneZahrady from '../schemas/komunitneZahrady'
import { FormDefinition, FormDefinitionType } from './formDefinitionTypes'

export const formDefinitions: FormDefinition[] = [
  {
    type: FormDefinitionType.SlovenskoSkGeneric,
    slovenskoSkUrl:
      'https://formulare.slovensko.sk/_layouts/eFLCM/DetailVzoruEFormulara.aspx?vid=00603481.stanoviskoKInvesticnemuZameru&vh=0&vl=8',
    slug: 'stanovisko-k-investicnemu-zameru',
    title: 'Žiadosť o stanovisko k investičnému zámeru',
    schemas: stanoviskoKInvesticnemuZameru,
    pospID: '00603481.stanoviskoKInvesticnemuZameru',
    pospVersion: '0.8',
    gestor: 'Martin Pinter',
    termsAndConditions: generalTermsAndConditions,
    messageSubjectDefault: 'Podanie',
    messageSubjectFormat:
      'e-ZST ž. {stavba.ulica} {stavba.nazov}, p.č. {stavba.parcelneCislo} kú {stavba.kataster}',
    ginisAssignment: {
      ginisOrganizationName: 'OUIC',
      ginisPersonName: 'Vícenová Marcela',
    },
    isSigned: false,
  },
  {
    type: FormDefinitionType.SlovenskoSkGeneric,
    slovenskoSkUrl:
      'https://formulare.slovensko.sk/_layouts/eFLCM/DetailVzoruEFormulara.aspx?vid=00603481.zavazneStanoviskoKInvesticnejCinnosti&vh=0&vl=7',
    slug: 'zavazne-stanovisko-k-investicnej-cinnosti',
    title: 'Žiadosť o záväzné stanovisko k investičnej činnosti',
    schemas: zavazneStanoviskoKInvesticnejCinnosti,
    pospID: '00603481.zavazneStanoviskoKInvesticnejCinnosti',

    pospVersion: '0.7',
    gestor: 'Martin Pinter',
    termsAndConditions: generalTermsAndConditions,
    messageSubjectDefault: 'Podanie',
    messageSubjectFormat:
      'e-SIZ ž. {stavba.ulica} {stavba.nazov}, p.č. {stavba.parcelneCislo} kú {stavba.kataster}',
    ginisAssignment: {
      ginisOrganizationName: 'OUIC',
      ginisPersonName: 'Vícenová Marcela',
    },
    isSigned: false,
  },
  {
    type: FormDefinitionType.SlovenskoSkGeneric,
    slovenskoSkUrl:
      'https://formulare.slovensko.sk/_layouts/eFLCM/DetailVzoruEFormulara.aspx?vid=00603481.predzahradky&vh=1&vl=0',
    slug: 'predzahradky',
    title: 'Predzáhradky',
    schemas: predzahradky,
    pospID: '00603481.predzahradky',
    pospVersion: '1.0',
    gestor: 'Martin Pinter',
    termsAndConditions: generalTermsAndConditions,
    messageSubjectFormat: 'Registrácia - {predzahradka.typRegistracie} predzáhradka',
    messageSubjectDefault: 'Registrácia predzáhradky',
    isSigned: false,
    ginisAssignment: {
      ginisOrganizationName: 'OUIC',
      ginisPersonName: 'Vícenová Marcela',
    },
  },
  {
    type: FormDefinitionType.SlovenskoSkGeneric,
    slovenskoSkUrl:
      'https://formulare.slovensko.sk/_layouts/eFLCM/DetailVzoruEFormulara.aspx?vid=00603481.komunitneZahrady&vh=1&vl=0',
    slug: 'komunitne-zahrady',
    title: 'Komunitné záhrady',
    schemas: komunitneZahrady,
    pospID: '00603481.komunitneZahrady',
    pospVersion: '1.0',
    gestor: 'Martin Pinter',
    termsAndConditions: generalTermsAndConditions,
    messageSubjectDefault: 'Žiadosť o komunitnú záhradu',
    messageSubjectFormat: 'Žiadosť o komunitnú záhradu - {pozemok.typPozemku} mestský pozemok',
    isSigned: false,
    ginisAssignment: {
      ginisOrganizationName: 'OUIC',
      ginisPersonName: 'Vícenová Marcela',
    },
  },
  {
    type: FormDefinitionType.SlovenskoSkTax,
    slovenskoSkUrl:
      'https://formulare.slovensko.sk/_layouts/eFLCM/DetailVzoruEFormulara.aspx?vid=esmao.eforms.bratislava.obec_024&vh=201501&vl=2',
    title: 'Priznanie k dani z nehnuteľností',
    slug: 'priznanie-k-dani-z-nehnutelnosti',
    schemas: priznanieKDaniZNehnutelnosti,
    pospID: 'esmao.eforms.bratislava.obec_024',
    pospVersion: '201501.2',
    gestor: 'Martin Pinter',
    termsAndConditions: taxTermsAndConditions,
    messageSubjectDefault: 'Priznanie k dani z nehnuteľností',
    isSigned: true,
  },
]
