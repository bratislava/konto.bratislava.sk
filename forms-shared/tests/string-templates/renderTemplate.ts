import {
  renderFormAdditionalInfo,
  renderFormTemplate,
} from '../../src/string-templates/renderTemplate'
import { FormDefinition } from '../../src/definitions/formDefinitionTypes'

describe('renderFormTemplate', () => {
  it('should render a simple template', () => {
    const formData = { name: 'John', age: 30 }
    const templateString = 'Name: <%= it.formData.name %>, Age: <%= it.formData.age %>'
    const result = renderFormTemplate(formData, templateString)
    expect(result).toBe('Name: John, Age: 30')
  })

  it('should use helper functions', () => {
    const formData = { numbers: [1, 2, 3] }
    const templateString = 'Numbers: <%= it.helpers.safeArray(it.formData.numbers).join(", ") %>'
    const result = renderFormTemplate(formData, templateString)
    expect(result).toBe('Numbers: 1, 2, 3')
  })

  it('should return null on error', () => {
    const formData = {}
    const templateString = '<%= nonExistentFunction() %>'
    const result = renderFormTemplate(formData, templateString)
    expect(result).toBeNull()
  })

  it('should return null when rendered string is empty', () => {
    const formData = { emptyArray: [] }
    const templateString = '<%= it.formData.emptyArray.join(", ") %>'
    const result = renderFormTemplate(formData, templateString)
    expect(result).toBeNull()
  })

  it('should return null when rendered string contains only whitespace', () => {
    const formData = {}
    const templateString = '  \n\t  '
    const result = renderFormTemplate(formData, templateString)
    expect(result).toBeNull()
  })
})

describe('renderFormAdditionalInfo', () => {
  it('should return null if no additionalInfoTemplate', () => {
    const formDefinition = {
      additionalInfoTemplate: undefined,
    } as FormDefinition
    const formData = {}
    const result = renderFormAdditionalInfo(formDefinition, formData)
    expect(result).toBeNull()
  })

  it('should render additional info template', () => {
    const formDefinition = {
      additionalInfoTemplate: 'Additional info: <%= it.formData.info %>',
    } as FormDefinition
    const formData = { info: 'Some extra information' }
    const result = renderFormAdditionalInfo(formDefinition, formData)
    expect(result).toBe('Additional info: Some extra information')
  })
})
