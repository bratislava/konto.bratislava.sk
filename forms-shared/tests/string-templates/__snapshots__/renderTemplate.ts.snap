// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderFormAdditionalInfo ziadostONajomBytuExample rendered template should match snapshot 1`] = `
"### Zoznam potrebných dokumentov

#### Žiadateľ/žiadateľka

- Osobné údaje - občiansky preukaz
- Osobné údaje - rozsudok o rozvode, sobášny list alebo iný doklad dokazujúci rodinný stav
- Adresa - dokumenty potvrdujúce pôsobenie v Bratislave, napr. pracovnú zmluvu, nájomnú zmluvu, potvrdenie o návšteve školy, potvrdenie z ubytovne, nocľahárne, potvrdenie sociálneho pracovníka o kontakte s klientom a pod.
- Súčasné bývanie - doklad o budúcom prepustení z ÚVTOS alebo resocializačného zariadenia
- Príjem - dokumenty dokazujúce všetky príjmy, napr. doklad od zamestnávateľa, potvrdenie z daňového úradu, potvrdenie o výške dôchodku, doklad o poberaní prídavkov na dieťa/deti, o poberaní materského, rodičovský príspevok, doklad o určení výšky výživného a pod.
- Zdravotný stav - dokumenty dokazujúce zdravotný stav, napr. potvrdenie o chronickom ochorení od ošetrujúceho lekára
- Rizikové faktory - prepúšťacia správa zo zariadenia ústavnej starostlivosti

#### Druh/družka

- Osobné údaje - kópia občianskeho preukazu
- Príjem - dokumenty dokazujúce všetky príjmy druha/družky, napr. doklad od zamestnávateľa, potvrdenie z daňového úradu, potvrdenie o výške dôchodku, doklad o poberaní prídavkov na dieťa/deti, o poberaní materského, rodičovský príspevok, doklad o určení výšky výživného a pod.
- Zdravotný stav - dokumenty dokazujúce zdravotný stav druha/družky, napr. potvrdenie o chronickom ochorení od ošetrujúceho lekára

#### Nezaopatrené deti do 25 rokov

##### Dieťa č. 1 (Lucia Nováková)
- Osobné údaje - kópia rodného listu dieťaťa, resp. kópia občianskeho preukazu, ak už dieťa dovŕšilo vek 15 rokov
- Príjem - potvrdenie o návšteve školy
##### Dieťa č. 2 (Michal Novák)
- Osobné údaje - kópia rodného listu dieťaťa, resp. kópia občianskeho preukazu, ak už dieťa dovŕšilo vek 15 rokov
- Príjem - potvrdenie o návšteve školy
- Príjem - potvrdenie o príjme dieťaťa
- Zdravotný stav - dokumenty dokazujúce uvedený zdravotný stav dieťaťa, napr. potvrdenie o chronickom ochorení od ošetrujúceho lekára

#### Iní členovia/členky domácnosti

##### Člen/členka domácnosti č. 1 (Mária Kováčová)
- Osobné údaje - kópia občianskeho preukazu
- Osobné údaje - doklad dokazujúci rodinný stav člena/členky domácnosti
- Príjem - dokumenty dokazujúce všetky príjmy člena/členky domácnosti, napr. doklad od zamestnávateľa, potvrdenie z daňového úradu, potvrdenie o výške dôchodku, doklad o poberaní prídavkov na dieťa/deti, o poberaní materského, rodičovský príspevok, doklad o určení výšky výživného a pod.
- Zdravotný stav - dokumenty dokazujúce zdravotný stav člena/členky domácnosti, napr. potvrdenie o chronickom ochorení od ošetrujúceho lekára
"
`;
